Makefile

```
@@ -52,7 +55,7 @@ docs: docs/node_modules
 
 .PHONY: lint
 lint: .bin/golangci-lint
-		golangci-lint run -v ./...
+		golangci-lint run -v --timeout 5m ./...
```

This can sometimes be very slow...

```suggestion
		golangci-lint run -v --timeout 10m ./...
```

---

done